apps/andi/test/integration/andi_web/live/extract_http_step_form_test.exs

```diff
@@ -1,10 +1,12 @@
-defmodule AndiWeb.ExtractHttpStepFormTest do
+defmodule AndiWeb.ExtractHttpStepTest do
```

Comment:
Oh, that's maybe why they were duplicated?

Reply:
Could be, yeah